libs/authentication/api/src/lib/auth-session.ts

```diff
@@ -1,20 +1,20 @@
 import {User, UserRole} from '@prisma/client'
 
-export enum SessionType {
+export enum AuthSessionType {
```

Comment:
I was actually ok with just `Session`.  Session could hold different data for different reasons. Just how it's used later determines if it's Auth piece.

Reply:
Can still be `Auth`, was just curious about your reasoning.

Reply:
I renamed to be with Auth as we have also `Session` in the database and it was confusing which one is which